src/main/java/uk/gov/hmcts/darts/log/service/impl/ArmLoggerServiceImpl.java

```diff
@@ -38,4 +38,25 @@ public void archiveToArmSuccessful(Integer eodId) {
     public void archiveToArmFailed(Integer eodId) {
         log.info(ARCHIVE_TO_ARM_FAILED, eodId);
     }
+
+    @Override
+    public void armRpoSearchSuccessful(Integer executionId) {
+        log.info("ARM RPO Search - Successfully completed for execution Id = {}");
```

Comment:
Worth adding some unit tests for this, similar to the tests that exist in `src/test/java/uk/gov/hmcts/darts/log/service/impl`, which use a log captor to verify the logging statements are as expected. Basically if this line is ever changed, it will break integration with Dynatrace, so unit tests will serve to flag any regressions here.

Reply:
done